packages/kbn-openapi-generator/redocly_linter/config.yaml

```diff
@@ -0,0 +1,25 @@
+plugins:
```

Comment:
Nit: Would be great here to add a comment with some links to the recommended rules and how to extend redocly with custom plugins.